docs/systems/servers.md

```diff
@@ -44,7 +44,16 @@ Old pages are redirected by Apache (search for `extensions.open-contracting.org`
 
 ## Kingfisher
 
-* `scrape.kingfisher.open-contracting.org`
-* `archive.kingfisher.open-contracting.org`
+OCDS Kingfisher is the data access and analysis platform for OCDS data. 
+
+There is a hosted version of the software, called Hosted Kingfisher. Documentation can be found at https://ocdskingfisher.readthedocs.io/en/latest/ 
```

Comment:
Please hyperlink "Hosted Kingfisher" and remove the second sentence, since we try to avoid bare links.